lib/css/modules/_showImages.scss

```diff
@@ -38,6 +39,22 @@ img {
 		padding: 0;
 	}
 
+	&-requires-permission {
```

Comment:
Does this need to be in CSS? Ideally the lock icon would be directly in the spritesheet.

If there's nothing I'm missing, this spritesheet should work:

![expandobuttons](https://cloud.githubusercontent.com/assets/7673145/18228972/d0b8ed56-7232-11e6-8da7-019bcff4ec9a.png)